### PR TITLE
Update portal with EVM Docs

### PR DESCRIPTION
**Pull request by assetcorp**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -312,6 +312,10 @@ const config = {
                   label: "Rust Docs",
                   href: "/docs/current/developer-docs/backend/rust/?source=nav",
                 },
+                {
+                  label: "Solidity Docs",
+                  href: "https://bitfinity.network/ecosystem/solidity-examples",
```
Comment (blutooth) [inline docusaurus.config.js:317]: This should link here: https://bitfinity.network/ecosystem/solidity-examples

Comment (assetcorp) [reply docusaurus.config.js]: I have updated this.

File: docusaurus.config.js [hunk 2]
```diff
@@ -312,6 +312,10 @@ const config = {
                   label: "Rust Docs",
                   href: "/docs/current/developer-docs/backend/rust/?source=nav",
                 },
+                {
+                  label: "Solidity Docs",
+                  href: "https://docs.bitfinity.network/",
+                },
```
[the remark this answers is not in the document]
Comment (sesi200) [reply docusaurus.config.js:318]: From internal discussion:

> TLDR: Yes, i support it so long as we know and tested their stuff actually works...

I had a look through your docs and I'm not sure how much confidence I have so far. Your FAQ [says](https://docs.bitfinity.network/faq#how-do-i-get-started-with-bitfinity) "The Bitfinity EVM testnet has been deployed on the Internet Computer. It is currently under development". Does that mean it's not functional yet? Also, your [ecosystem](https://bitfinity.network/ecosystem) page doesn't really link to Bitfinity projects AFAIU. And IC integration [seems to be under development](https://docs.bitfinity.network/bridge/transfer-bitfinity-ic).

From that I don't think it's ready for a full-blown main nav entry. Do you think that's a fair assessment or did I miss something big?

[the remark this answers is not in the document]
Comment (blutooth) [reply docusaurus.config.js:318]: It is functional, we have had dapps deployed to the EVM. It is under development, however. Check out this third party app from our ecosystem page, which is deployed to the testnet: https://rsusi-aqaaa-aaaag-abhda-cai.ic0.app/#/

[the remark this answers is not in the document]
Comment (blutooth) [reply docusaurus.config.js:318]: I think this is unfair, even though  tokens between IC and the EVM are under development, users can still deploy apps to the EVM on the IC.

[the remark this answers is not in the document]
Comment (blutooth) [reply docusaurus.config.js:318]: Also bridges can be deployed by third-parties and are not core to the EVM engine.

[the remark this answers is not in the document]
Comment (blutooth) [reply docusaurus.config.js:318]: Lastly, it is well tested with 99% coverage and increasing on a suite of behavioural tests.

File: src/components/DocsHome/Languages/index.tsx [hunk 3]
```diff
@@ -82,8 +83,23 @@ function Index() {
...
+        <motion.a
+          variants={transitions.item}
+          href={"https://bitfinity.network/ecosystem/solidity-examples"}
+          target={"_blank"}
+          className={clsx(styles.card, styles.cardHover)}
+        >
+          <div className={styles.bodyContainer}>
+            <p className={styles.informationTitle}>Solidity</p>
+            <p className={styles.informationBody}>
+              The Bitfinity EVM allows you to run Solidity smart-contracts on the Internet Computer
```
Comment (sesi200) [inline src/components/DocsHome/Languages/index.tsx:98]:
```suggestion
              The Bitfinity EVM allows you to run Solidity smart contracts on the Internet Computer
```
according to the new style guide

**Review by sesi200 — APPROVED** [review]
